src/tools/tidy/src/ui_tests.rs

```diff
@@ -10,8 +10,10 @@ use std::path::{Path, PathBuf};
 
 const ENTRY_LIMIT: usize = 900;
 // FIXME: The following limits should be reduced eventually.
-const ISSUES_ENTRY_LIMIT: usize = 1893;
-const ROOT_ENTRY_LIMIT: usize = 872;
+// Max number of files under "./ui/issues"
+const ISSUES_ENTRY_LIMIT: usize = 1892;
+// Max number of files under "./ui"
+const ROOT_ENTRY_LIMIT: usize = 871;
```

Comment:
Ideally I shouldnt need to touch this

Reply:
Unless maybe you're moving things out of this folder. Otherwise, you are already adding a tests in a subfolder, so things should be fine from that point.